Cargo.toml

```diff
@@ -25,7 +25,7 @@ branch = "current"
 features = ["builtin-queue", "yt-dlp"]
 
 [dependencies.serenity]
-version = "0.10"
+version = "0.10.10"
```

Comment:
Weird, `0.10` should fetch the latest patch version, right? Which would make it `0.10.10`? Have you tried with just `0.10` but making `cargo clean` and then `cargo run` ?

Reply:
Can confirm, just tested and all is working when when keeping `0.10`.

Reply:
Missed the `cargo clean` rolling it back!

Reply:
For some reason, even after a `cargo clean` it uses `v0.10.9`. Did you try `cargo clean` in this branch @aquelemiguel?